src/main/scala/org/renci/cam/Server.scala

```diff
@@ -53,7 +56,7 @@ object Server extends App {
         servr <-
           BlazeServerBuilder[Task](runtime.platform.executor.asEC)
             .bindHttp(appConfig.port, appConfig.host)
-            .withHttpApp(httpAppWithLogging)
+            .withHttpApp(CORS.httpApp(httpAppWithLogging))
```

Comment:
I think here you can just use `CORS(httpAppWithLogging)`. Not sure that it matters at all.